src/goto-instrument/Makefile

```diff
@@ -12,6 +13,7 @@ SRC = accelerate/accelerate.cpp \
       accelerate/scratch_program.cpp \
       accelerate/trace_automaton.cpp \
       accelerate/util.cpp \
+      abstraction_spect.cpp \
```

Comment:
Similar.

Reply:
Renamed to rr-abstraction-spect.cpp

Reply:
Great; thank you.  I appreciate that this is kinda tedious but there are at least ... 4, maybe more, different kinds of abstraction in the code-base already.